data_import/views.py

```diff
@@ -36,17 +38,14 @@ def get_and_log(self, request, key_object=None):
                 "access_token": key_object.access_token,
                 "project_id": key_object.project_id,
             }
-        else:
-            key = {}
-        access_log = NewDataFileAccessLog(
-            user=user,
-            ip_address=get_ip(request),
-            data_file=self.data_file,
-            data_file_key=json.dumps(key),
         )
-        access_log.save()
 
-        return HttpResponseRedirect(self.data_file.file_url_as_attachment)
+        aws_url = self.data_file.file_url_as_attachment
+        access_log.aws_url = aws_url
+
+        url = "{0}&x-oh-key={1}".format(aws_url, key_object.key)
```

Comment:
This is minor but, if this is what's returned, should it also be what's stored in the `aws_url` as well?